src/app/parcels/form/ParcelForm.tsx

```diff
@@ -255,6 +266,7 @@ const ParcelForm: React.FC<ParcelFormProps> = ({
 
         const parcelRecord = {
             client_id: clientId || fields.clientId || "",
+            list_type: fields.listType as ListType,
```

Comment:
This cast seems a bit dangerous? The compiler thinks this can be null, which would cause submission errors, but we're insisting it's not. This feels like it should be handled by the submission checks, in which case we can make sure it's not null and maintain type safety, or we can just default it to `default` and never let it be null in the first place.